securedrop/crypto_util.py

```diff
@@ -49,7 +50,7 @@ def do_runtime_tests():
 gpg = gnupg.GPG(binary='gpg2', homedir=config.GPG_KEY_DIR)
 
 # map code for a given language to a localized wordlist
-language2words = {}
+language2words = {}  # type: Dict[Text,List[Text]]
```

Comment:
I think it would be better to limit the type declaration to something the static anlysis can actually verify, for instance:
<pre>
language2words = {}  # type: Dict[Any,Any]
</pre>
instead of what we know to be true at this moment in time. This will help maintainability. For instance, if the data structure changes in the future, there is a good chance this declaration will be overlooked and become an incorrect and confusing description.

Reply:
1. Usage of `Any` is the last option. We should try to avoid it as much as possible.
2. In future if we change the type, `mypy` will catch that and fail (error). That is the reason why use type annotation checking in CI.

Reply:
Hum, I may have missed something then. If I change `type: Dict[Text,List[Text]]` to `type: Dict[int,int]` it does not throw an error. Why is that ?

Reply:
@dachary Because we are yet to add type annotations in https://github.com/freedomofpress/securedrop/blob/8d6f9db6a6fbb1a33cf1324828de7c99e2ccb12c/securedrop/crypto_util.py#L83 mypy by design will not check/verify the usage unless the calling function is also typed. 

This enables us to have incremental type annotations. 

If you want to experiment, just add this in the line below that type annotation.

```
language2words[42] = 9001
```

Reply:
Oh, I see. Would you mind adding annotations to **_get_wordlist** so it is properly enforced ? That would be a good example of a minimal, consistent use of mypy. 👍